manifests/init.pp

```diff
@@ -96,4 +96,8 @@
     ensure => symlink,
     target => $hiera_yaml,
   }
+  # Restart master service
+  Service <| title == $hiera::params::master_service |> {
+    subscribe +> File[$hiera_yaml],
+  }
```

Comment:
Yes, collector won't collect anything, since it won't find service in the
catalog


On Fri, Aug 7, 2015 at 10:38 AM, Igor Galić <notifications@github.com>
wrote:

> In manifests/init.pp
> <https://github.com/hunner/puppet-hiera/pull/71#discussion_r36524825>:
>
> > @@ -96,4 +96,8 @@
> >      ensure => symlink,
> >      target => $hiera_yaml,
> >    }
> > +  # Restart master service
> > +  Service <| title == $hiera::params::master_service |> {
> > +    subscribe +> File[$hiera_yaml],
> > +  }
>
> will this work if there's no master on the node?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hunner/puppet-hiera/pull/71/files#r36524825>.
>